spec/concepts/stock/task/import_csv_spec.rb

```diff
@@ -5,74 +5,45 @@
 
   subject { described_class.call csv: csv, model: model, logger: logger }
 
-  let(:logger) { instance_spy Logger }
-
-  shared_examples 'importing a valid CSV' do
-    it { is_expected.to be_success }
-
-    it 'imports all' do
-      expect { subject }.to change(model, :count).by 3
+  class RandomModel
+    def self.header_mapping
+      { test: :hello_world }
     end
+  end
 
-    it 'read the file by chunk' do
-      expect(SmarterCSV)
-        .to receive(:process)
-        .with(csv, include(chunk_size: 2_000))
-
-      subject
+  let(:logger) { instance_spy Logger }
+  let(:model) { RandomModel }
```

Comment:
Ici, purement pour éviter de créer un nom de classe (même si il y a effectivement peu de chance que ça rentre en conflit avec un autre composant du même nom) moi je fais ça : 

```ruby
let(:model) do 
  Class.new do 
    self.header_mapping
      { test: :hello_world } 
    end 
  end 
end 
```

T'as une classe sans nom, which is nice ^^